pontoon/base/static/js/manage_permissions.js

```diff
@@ -102,7 +107,7 @@ $(function() {
 
     // Update menu (must be above Copying Translators)
     $(this).addClass('hidden');
-    if ($('#project-selector .menu li:visible').length === 0) {
+    if ($('#project-selector .menu li:not(".hidden")').length === 0) {
```

Comment:
The `:visible` selector wasn't reliable - if you searched for a project, items could get hidden, so if only one project matched the criteria and you added it, the selector disappeared.